src/templates/_elements/sources.twig

```diff
@@ -38,7 +38,13 @@
     {% endif %}
     <span class="label">{{ source.label|trim is not same as('') ? source.label : '(blank)'|t('app') }}</span>
     {% if source.badgeCount is defined %}
-        <span class="badge">{{ source.badgeCount|number(decimals=0) }}</span>
+        <span class="badge">
+            {{ source.badgeCount|number(decimals=0) }}
+            {{ tag('span', {
+                class: 'visually-hidden',
+                text: source.badgeCount == 1 ? 'Notification'|t('app') : 'Notifications'|t('app')
+            }) }}
+        </span>
```

Comment:
Ideally this should be done in a way where the visually-hidden text completely overrides the visible text, so that the number and “notification(s)” can be combined into a single translated string, using the following syntax:

```twig
{{ tag('span', {
    class: 'visually-hidden',
    text: '{num, number} {num, plural, =1{notification} other{notifications}}'|t('app', {
        num: source.badgeCount,
    }),
}) }}
```

Maybe we can add a new `no-speak` class with `speak: none` (if there’s not something already like that – I forget), and then wrap the initial `{{ source.badgeCount|number(decimals=0) }}` tag in a `<span class="no-speak">`.